src/AccessibilityInsights.SharedUx/KeyboardHelpers/HotKey.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Win32;
+using System;
+using System.Linq;
+using System.Windows.Forms;
+
+namespace AccessibilityInsights.SharedUx.KeyboardHelpers
+{
+    /// <summary>
+    /// Class for hot key
+    /// </summary>
+    public class HotKey
+    {
+        /// <summary>
+        /// this value will be set by Hotkey Handler
+        /// </summary>
+        public int Id { get; internal set; }
+
+        public Keys Key { get; internal set; }
+        public HotkeyModifier Modifier { get; internal set; }
+
+        public Action Action { get; set; }
+
+        public uint ErrorCode { get; private set; }
+
+        /// <summary>
+        /// Register hot key
+        /// </summary>
+        /// <param name="hWnd"></param>
+        public bool Register(IntPtr hWnd)
+        {
+            this.ErrorCode = 0;
+
+            if(NativeMethods.RegisterHotKey(hWnd, Id, (int)this.Modifier, Key.GetHashCode()) == false)
+            {
+                ErrorCode = NativeMethods.GetLastError();
+                return false;
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Unregister HotKey
+        /// </summary>
+        /// <param name="hWnd"></param>
+        public bool Unregister(IntPtr hWnd)
+        {
+            this.ErrorCode = 0;
+
+            if(NativeMethods.UnregisterHotKey(hWnd, Id) == false)
+            {
+                ErrorCode = NativeMethods.GetLastError();
+                return false;
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Get an instance of HotKey based on given string
+        /// </summary>
+        /// <param name="txt"></param>
+        /// <returns></returns>
+        public static HotKey GetInstance(string txt)
+        {
+            var atoms = from a in txt.Split(new char[] { '+' }, StringSplitOptions.RemoveEmptyEntries)
+                        select a.Trim();
+            var hk = new HotKey();
+
+            try
+            {
+                if (atoms.Count() == 2)
+                {
+                    hk.Modifier = GetMofidifier(atoms.ElementAt(0));
+
```

Comment:
the UI makes it seem like there is an extra blank line here. If that is true, can we remove it?